extensions/sql-database-projects/src/models/tree/databaseReferencesTreeItem.ts

```diff
@@ -19,7 +19,7 @@ export class DatabaseReferencesTreeItem extends BaseProjectTreeItem {
 	private references: DatabaseReferenceTreeItem[] = [];
 
 	constructor(project: ProjectRootTreeItem) {
-		super(vscode.Uri.file(path.join(project.uri.path, constants.databaseReferencesNodeName)), project);
+		super(vscode.Uri.file(path.join(path.parse(project.projectUri.fsPath).base, constants.databaseReferencesNodeName)), project);
```

Comment:
ah this change isn't necessary anymore after I changed the ProjectRootTreeItem's projectUri back to just the basename instead of the file system uri. Is it simple enough to keep it here now that the extra path.parse().base is gone or would it be cleaner to move it out?